internal/extproc/translator/openai_awsbedrock_test.go

```diff
@@ -596,6 +871,49 @@ func TestOpenAIToAWSBedrockTranslatorV1ChatCompletion_ResponseBody(t *testing.T)
 				},
 			},
 		},
+		{
+			name: "test tool use",
+			input: awsbedrock.ConverseOutput{
+				Output: &awsbedrock.ConverseOutput_{
+					Message: awsbedrock.Message{
+						Role: awsbedrock.ConversationRoleAssistant,
+						Content: []*awsbedrock.ContentBlock{
+							{
+								Text: ptr.To("response"),
+								ToolUse: &awsbedrock.ToolUseBlock{
+									Name:      "exec_python_code",
+									ToolUseID: "call_6g7a",
+									Input:     "{\"code_block\":\"from playwright.sync_api import sync_playwright\\n\"}}",
+								},
+							},
+						},
+					},
+				},
+			},
+			output: openai.ChatCompletionResponse{
+				Object: "chat.completion",
+				Choices: []openai.ChatCompletionResponseChoice{
+					{
+						Index:        0,
+						FinishReason: openai.ChatCompletionChoicesFinishReasonStop,
```

Comment:
The finish reason here should be “tool_calls” as the stop reason on ConverseOutput is set to toolUse in this case.

Reply:
update the unit test, for input bedrock converse response, the stop reason is `tool_use` for content with toolUseBlock